README.md

```diff
@@ -1,50 +1,39 @@
 # CRS Documentation Repository
 
-This repository contains the documentation for OWASP CRS.
+This repository contains the documentation for OWASP CRS. For the official website, see [https://coreruleset.org/](https://coreruleset.org/).
 
-## For users of the CRS
+## For Readers
 
-The generated documentation is automatically updated at https://coreruleset.org/docs/. If you just want to read the documentation, you can find it there. If you intend to _contribute_ to the documentation, the rest of this page will guide you through it.
+The generated documentation is automatically updated at https://coreruleset.org/docs/. If you just want to read the documentation, you can find it there.
 
-## Requirements
+## For Contributors
 
-You can edit the documentation on your local system. The only thing you need is the latest [Hugo binary](https://gohugo.io/getting-started/installing/) for your OS (Windows, Linux, Mac): it’s that simple.
+Welcome! This page will guide you through the process of contributing to the CRS documentation. Before you start, please read our [Contribution Guideline](content/development/contribution_guidelines.md).
 
-**Important: You need Hugo _extended_ version >= 0.93.0.**
+### 1. Prerequisites
 
-## Cloning this repository
+To contribute to the CRS documentation, The only thing you need is the latest [Hugo binary](https://gohugo.io/getting-started/installing/) for your OS (Windows, Linux, Mac) **(Important: You need Hugo _extended_ version >= 0.93.0.)**.
```

Comment:
```suggestion
To contribute to the CRS documentation the only thing you need is the latest [Hugo binary](https://gohugo.io/getting-started/installing/) for your OS (Windows, Linux, Mac) **(Important: You need Hugo _extended_ version >= 0.93.0.)**.
```

Reply:
Fixed in [f8a8b1d](https://github.com/coreruleset/documentation/pull/132/commits/f8a8b1d136f7c5ab726e33c8fab633d945b6dbfd).